src/api/sso/src/middleware.js

```diff
@@ -163,6 +164,11 @@ module.exports.createTelescopeUser = function createTelescopeUser() {
       next(createError(status, 'Unable to create a Telescope profile'));
       return;
     }
+    try {
+      await publishSignUpMessage(req.body.displayName, req.body.githubUsername, req.body.blogUrl);
```

Comment:
We could actually **not** wait on this, and it can return immediately, since the rest of the code doesn't depend on the outcome:

```js
publishSignUpMessage(req.body.displayName, req.body.githubUsername, req.body.blogUrl)
  .catch(error => logger.warn({ error } , 'Unable to publish Slack signup message');
next();
```

Reply:
Resolved in [7f17ce0](https://github.com/Seneca-CDOT/telescope/pull/3773/commits/7f17ce0cb32cb469696e4a53bff8ed52589a9a45).